steal.lic

```diff
@@ -153,7 +153,8 @@ class Steal
     {
       'new' => true, 'room' => Room.current.id, 'province' => 'zoluren',
```

Comment:
I was wondering if we should keep province as you mentioned below since it is such a board area.  When my thief is in Muspar'i, I wouldn't want him trying to run to Riverhaven to steal.  I left it the same because it wasn't doing anything.